x-pack/plugins/apm/server/lib/correlations/get_correlations_for_slow_transactions/index.ts

```diff
@@ -87,7 +96,20 @@ export async function getCorrelationsForSlowTransactions({
                 significant_terms: {
                   size: 10,
                   field: fieldName,
-                  background_filter: { bool: { filter: backgroundFilters } },
+                  background_filter: {
+                    bool: {
+                      filter: [
+                        ...backgroundFilters,
+                        {
+                          range: {
+                            [TRANSACTION_DURATION]: {
+                              lt: durationForPercentile,
+                            },
```

Comment:
Isn't the foreground supposed to be a subset of the background?

Reply:
... if not you may have to set this 
```ts
"background_is_superset": false
```

Reply:
We found that the existing background filter included too much uninteresting data and it was skewing the results of significant terms agg to be not as helpful see(https://youtu.be/azP15yvbOBA)